scripts/TPI.py

```diff
@@ -47,7 +55,7 @@ def solve_tp(g_n_path, omega_S_preTP, rho_s, imm_rates_path, params):
         for t in range(1, T + 1):
```

Comment:
Sorry I meant here for the t loop
I think we can do the same here for n when we solve for the full life decision maker